src/main/java/com/volunteer/enterprise/VolunteerAppController.java

```diff
@@ -1,86 +1,72 @@
 package com.volunteer.enterprise;
 import com.volunteer.enterprise.dto.Company;
 import com.volunteer.enterprise.dto.Volunteer;
+import com.volunteer.enterprise.service.ICompanyService;
 import com.volunteer.enterprise.service.IVolunteerService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.ui.Model;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
 
 import java.util.List;
 
 @Controller
 public class VolunteerAppController {
+    @Autowired
     IVolunteerService volunteerService;
 
+    @Autowired
+    ICompanyService companyService;
+
     /**
-     *RequestMapping for root (/) endpoint
-     * Create a Voluntier volunteer object then display start
+     * GetMapping for root(/) endpoint
+     * Create a Voluntier volunteer object and then display start
+     *
      * @return Voluntier index page
      */
-
-    @RequestMapping("/")
+    @GetMapping("/")
     public String index(Model model){
-
-    Volunteer volunteer = new Volunteer();
-    volunteer.setVolunteerID(1);
-    volunteer.setVolunteerUser("Iron Man");
-    volunteer.setVolunteerPass("1234");
-    model.addAttribute(volunteer);
+        Volunteer volunteer = new Volunteer();
+        volunteer.setVolunteerID(1);
+        volunteer.setVolunteerEmail("tony@starkindustries.com");
+        volunteer.setVolunteerUser("Iron Man");
+        volunteer.setVolunteerPass("1234");
 
         // Add newly created favorite to model so it can be displayed
         model.addAttribute(volunteer);
         return "Index";
-<<<<<<< HEAD
-
     }
 
-
-
-
     /**
-     *RequestMapping for root (/) endpoint
-     * Create a Company company object then display company form page
+     * GetMapping for /company endpoint
+     * Create a company object and then display company form page
+     *
      * @return company form page
      */
-
-    @RequestMapping("/company")
+    @GetMapping("/company")
```

Comment:
Changed `@RequestMapping` to the more specific `@GetMapping` where applicable. [Reference](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/bind/annotation/GetMapping.html)